packages/oc-server-compiler/package.json

```diff
@@ -22,6 +22,7 @@
   "license": "MIT",
   "dependencies": {
     "async": "^2.4.1",
+    "babel-core": "^6.26.3",
```

Comment:
why did you introduced babel-core here?

Reply:
Was a missing peer dependency of the babel loader